catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -43,12 +52,16 @@
 
     private boolean copyIngestedFiles = false;
 
-    private CamelContext camelContext;
+    private final CamelContext camelContext;
 
     private List<RouteDefinition> routeCollection;
 
     private List<String> attributeOverrides;
 
+    private final RetryPolicy retryPolicyForContent;
```

Comment:
`RetryPolicy` is not thread safe.  I don't think there is more than one thread calling waitForContentCamelComponent currently but it might be safer to create the policy in that method in case that changes in the future.

https://github.com/jhalterman/failsafe/issues/47